tests/system/action/user/test_update_saml_account.py

```diff
@@ -0,0 +1,70 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class UserUpdateSamlAccount(BaseActionTestCase):
+    def test_update_saml_account_correct(self) -> None:
+        self.set_models({"user/78": {"username": "111222333", "saml_id": "111222333"}})
```

Comment:
Set a different `username` and check that it doesn't changed

Reply:
Okay, done.